src/ckeditor.jsx

```diff
@@ -103,11 +127,13 @@ CKEditor.propTypes = {
 	onChange: PropTypes.func,
 	onInit: PropTypes.func,
 	onFocus: PropTypes.func,
-	onBlur: PropTypes.func
+	onBlur: PropTypes.func,
+	disabled: PropTypes.oneOfType( [ PropTypes.bool, PropTypes.string ] )
```

Comment:
I don't agree here. Why we should support multiple types here? Can't we just support only `boolean`?

Reply:
Why not? You can do `disabled="true"` (as I did before I discovered `disabled={true}`) and it still will work.

Reply:
But `disabled="true"` isn't the preferred way to pass a boolean option to react component as you find out yourself... IMO we should support more than one type if there're a concept and more logic behind the other types (like string and array of strings).

Reply:
Now the component will accept boolean value only.